swift/test/Ice/defaultValue/Test.ice

```diff
@@ -4,8 +4,8 @@
 
 #pragma once
 
-[["swift:class-resolver-prefix:IceDefaultValue",
-  "suppress-warning:deprecated"]] // For enumerator references
+[["swift:class-resolver-prefix:IceDefaultValue"]]
+[["suppress-warning:deprecated"]] // For enumerator references
```

Comment:
For suppress warning, we already use deprecated?

Reply:
Yep. Of course! See: https://github.com/zeroc-ice/ice/issues/2093#issuecomment-2088648887
And maybe weigh in on the rest of that comment.